configure.ac

```diff
@@ -1,8 +1,9 @@
 ##
 # Prologue
 ##
-AC_INIT([flux-core],
-        m4_esyscmd([test -n "$FLUX_VERSION" && printf $FLUX_VERSION || git describe --always | awk '/.*/ {sub(/^v/, ""); printf "%s",$1; exit}']))
+AC_INIT([flux-core], 0.46.0)
```

Comment:
This looks like a local change that was inadvertently committed.

Reply:
Yeah that's a mistake - that is the only way I can get it to build. Let me fix that and I'll just do the rebase too.

Reply:
okay fixed!